dev/encoder/README.md

```diff
@@ -0,0 +1,15 @@
+# ReadIncrementalEncoderPio
+Reading high Speed Incremental Encoders with PIO in Rpi Pico and C++
+
+You could just boot the .uf2 file to the pico with encoder connected to pins GPio 16 and 17 
```

Comment:
Is this Readme still accurate? Probably not right? Is it even needed? We have some long comment in the cpp file, either remove the readme or make that the readme?

Reply:
no it's not, this was the readme copied from the original example we used, but we can just delete the readme and rely on code comments